presto-jmx/src/main/java/com/facebook/presto/connector/jmx/JmxMetadata.java

```diff
@@ -114,33 +121,61 @@ private JmxTableHandle getJmxHistoryTableHandle(SchemaTableName tableName)
     private JmxTableHandle getJmxTableHandle(SchemaTableName tableName)
     {
         try {
-            String canonicalName = new ObjectName(tableName.getTableName()).getCanonicalName();
-            Optional<ObjectName> objectName = mbeanServer.queryNames(WILDCARD, null).stream()
-                    .filter(name -> canonicalName.equalsIgnoreCase(name.getCanonicalName()))
-                    .findFirst();
-            if (!objectName.isPresent()) {
+            String objectNamePattern = toPattern(tableName.getTableName()); //toLowerCase(ENGLISH));
+            List<ObjectName> objectNames = mbeanServer.queryNames(WILDCARD, null).stream()
+                    .filter(name -> name.getCanonicalName().toLowerCase(ENGLISH).matches(objectNamePattern))
```

Comment:
where do we enforce that pattern is lowercase? Is it lowercases somewhere before this method?

Reply:
This why there was a comment. It looks like it is handled in the planner somewhere. There is a test for that.

Reply:
Ok - so change the comment so it actually says what it is supposed to :)

Reply:
You may also add an checkArgument that it is lowercased already.

Reply:
nit: I would prefer `checkAregument` that tableName is lowercased already. 
When presto will convert to support case sensitive table names it will be easier to catch places which will need to be fixed.